server/statistics/hot_peer_cache_test.go

```diff
@@ -470,22 +498,9 @@ func (t *testHotPeerCache) TestRemoveFromCacheRandom(c *C) {
 
 func BenchmarkCheckRegionFlow(b *testing.B) {
 	cache := NewHotPeerCache(Read)
-	region := core.NewRegionInfo(&metapb.Region{
-		Id: 1,
-		Peers: []*metapb.Peer{
-			{Id: 101, StoreId: 1},
-			{Id: 102, StoreId: 2},
-			{Id: 103, StoreId: 3},
-		},
-	},
-		&metapb.Peer{Id: 101, StoreId: 1},
-	)
-	newRegion := region.Clone(
-		core.WithInterval(&pdpb.TimeInterval{StartTimestamp: 0, EndTimestamp: 10}),
-		core.SetReadBytes(30000*10),
-		core.SetReadKeys(300000*10))
+	region := buildRegion(Read, 3, 10)
```

Comment:
It changes the original value. Is it expected?

Reply:
It doesn't matter, just make sure it's hot